modules/testing/src/backend/api.ts

```diff
@@ -42,6 +43,16 @@ export default async (bp: typeof sdk, testByBot: TestByBot) => {
     res.sendStatus(200)
   })
 
+  router.post('/buildScenario', async (req, res) => {
+    try {
+      const { eventIds } = req.body
+      const builder = new Builder(bp.database, eventIds)
```

Comment:
do you event need a class for this ? do you plan to keep a state ? why not simply

```
const scenario = await buildScenarioFromEvents(eventIds, bp)
```

Reply:
Do you think we could we use the same "Builder" for live building ? or that's a really different concern and would mix unrealated logic ?

In this case, it would make sens to have a builder class